packages/manager/src/features/CloudPulse/shared/CloudPulseResourcesSelect.tsx

```diff
@@ -45,7 +45,13 @@ export const CloudPulseResourcesSelect = React.memo(
       disabled !== undefined ? !disabled : Boolean(region && resourceType),
       resourceType,
       {},
-      xFilter ? xFilter : { region }
+      resourceType === 'dbaas'
+        ? xFilter
+          ? { platform: 'rdbms-default', ...xFilter }
```

Comment:
always use spread operator first & then updated values. It can avoid loss in updated data

```suggestion
          ? { ...xFilter, platform: 'rdbms-default' }
```